sdcm/sct_runner.py

```diff
@@ -1060,7 +1060,9 @@ def update_sct_runner_tags(test_runner_ip: str = None, test_id: str = None, tags
         runner_to_update = [runner for runner in listed_runners if runner.test_id == test_id]
 
     if not runner_to_update:
-        raise RuntimeError(f"Could not find SCT runner with IP: {test_runner_ip} to update tags for.")
+        LOGGER.warning("Could not find SCT runner with IP: %s, test_id: %s to update tags for.",
```

Comment:
i would do it differently, as this function is trying to change tags on SCT Runner, but in case we don't have it, we should have nothing to do here..
so i would:
```
def update_sct_runner_tags(test_runner_ip: str = None, test_id: str = None, tags: dict = None):
    if not test_runner_ip:
        LOGGER.warning("Could not find SCT runner with IP: %s, test_id: %s to update tags for.",
        return

```

Reply:
We can find the test runner by ip or by test_id, so the check needs to be a bit after that